src/main/java/com/minecolonies/colony/CitizenData.java

```diff
@@ -427,9 +427,8 @@ public AbstractJob getJob()
         {
             return type.cast(job);
         }
-        catch(ClassCastException exc)
+        catch(ClassCastException ignored)
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1166)